docs/user/_index.md

```diff
@@ -1,76 +1,56 @@
 
 ## Metrics
 
-Unlike Prometheus, the Grafana Agent is _just_ targeting `remote_write`,
+Unlike Prometheus, the Grafana Agent _just_ targets `remote_write`,
 so some Prometheus features, such as querying, local storage, recording rules,
 and alerts aren't present. `remote_write`, service discovery, and relabeling
 rules are included.
 
 The Grafana Agent has a concept of an "instance", each of which acts as
 its own mini Prometheus agent with their own `scrape_configs` section and
 `remote_write` rules. More than one instance is useful when you want to have
-completely separate configs that write to two different locations without
+completely separate configurations that write to two different locations without
 needing to worry about advanced metric relabeling rules. Multiple instances also
 come into play for the [Scraping Service Mode]({{< relref "./scraping-service" >}}).
 
-The Grafana Agent can be deployed in three modes:
+The Grafana Agent for collecting metrics can be deployed in three modes:
 
 - Prometheus `remote_write` drop-in
 - [Host Filtering mode](#host-filtering)
-- [Scraping Service Mode]({{< relref "./scraping-service" >}})
+- [Scraping Service mode]({{< relref "./scraping-service" >}})
 
+### Prometheus `remote_write` drop-in
 The default deployment mode of the Grafana Agent is a _drop-in_
 replacement for Prometheus `remote_write`. The Agent will act similarly to a
 single-process Prometheus, doing service discovery, scraping, and remote
 writing.
 
-_Host Filtering mode_ is achieved by setting a `host_filter` flag on a specific
-instance inside the Agent's configuration file. When this flag is set, the
-instance will only scrape metrics from targets that are running on the same
-machine as the instance itself. This is extremely useful to migrate to sharded
-Prometheus instances in a Kubernetes cluster, where the Agent can be deployed as
-a DaemonSet and distribute memory requirements across multiple nodes.
-
-Note that Host Filtering mode and sharding your instances means that if an
-Agent's metrics are being sent to an alerting system, alerts for that Agent may
-not be able to be generated if the entire node has problems. This changes the
-semantics of failure detection, and alerts would have to be configured to catch
-agents not reporting in.
-
-The final mode, _Scraping Service Mode_ 
-clusters a subset of agents. It acts as a go-between for the drop-in mode
-(which does no automatic sharding) and `host_filter` mode (which forces sharding
-by node). The Scraping Service Mode clusters a set of agents with a set of
-shared configs and distributes the scrape load automatically between them. For
-more information, refer to ({{< relref "./scraping-service" >}}).
 
 ### Host filtering
-
 Host filtering configures Agents to scrape targets that are running on the same
-machine as the Grafana Agent process. It:
+machine as the Grafana Agent process.
 
 1. Gets the hostname of the agent by the `HOSTNAME` environment variable or
```

Comment:
I'm confused about using a numbered list here. Is there a 2. somewhere lower in the doc?

Reply:
yes